tools/dbtool.py

```diff
@@ -226,15 +239,15 @@ def fetch_configs():
         write_configs()
 
 def write_configs():
-    with open(r'config.yaml', 'w') as file:
+    with open(from_dbtool_path('config.yaml'), 'w') as file:
         dump = [{'mysql_bin' : mysql_bin}, {'auto_backup' : auto_backup}, {'auto_update_client' : auto_update_client}, {'db_ver' : db_ver}]
         yaml.dump(dump, file)
 
 def fetch_module_files():
-    with open('../modules/init.txt', 'r') as file:
+    with open(from_server_path('modules/init.txt'), 'r') as file:
         for line in file.readlines():
             if not line.startswith('#') and line.strip() and not line in ['\n', '\r\n']:
-                line = "../modules/" + line.strip()
+                line = from_server_path("modules/" + line.strip())
                 if pathlib.Path(line).is_dir():
                     for filename in pathlib.Path(line).glob('**/*.sql'):
                         import_files.append(str(filename).replace('\\', '/'))
```

Comment:
Think this needs path added too.

Reply:
`str(filename)` on a file object provides an absolute path, which is handy 👍. Confirmed with debugger